1-JavaScript/Week3/homework/ex4-shoppingCart.js

```diff
@@ -26,6 +26,13 @@ function addToShoppingCart(/* parameters go here */) {
 }
 
 // ! Test functions (plain vanilla JavaScript)
+function test0() {
```

Comment:
To be consistent with the other exercises I would suggest renumbering all tests so that this one can start as `test1()`.